CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+Contributing to Mypy
+====================
+
+Welcome!  Mypy is a community project that aims to work for a wide
+swath of Python users and Python codebases.  If you're trying Mypy on
+your Python code, your experience and what you can contribute is
+important to the project's success.
+
+
+Getting started, building, and testing
+--------------------------------------
+
+If you haven't already, take a look at the project's README file and
+the [Mypy documentation](http://mypy.readthedocs.org/en/latest/), and
+try adding type annotations to your file and type-checking it with Mypy.
+
+
+Discussion
+----------
+
+If you've run into behavior in Mypy you don't understand, or you're
```

Comment:
Without the "or" it becomes a bit of a garden-path sentence: "If you've blah blah blah, you've blah blah blah" sounds like the second clause is the "then" corresponding to the "if". The reader doesn't get a hint that it's actually part of a disjunctive list until many words later with the "or", and even then it's pretty ambiguous how it should parse. The fix of including this "or" is slightly inelegant but I think it's fine.